bundles/net.tourbook/src/net/tourbook/weather/WeatherUtils.java

```diff
@@ -87,12 +84,12 @@ public static String buildFullWeatherDataString(final float temperatureValue,
 
       final String precipitation = Messages.Log_HistoricalWeatherRetriever_001_WeatherData_Precipitation
             + UI.SPACE
-            + String.format("%5s", UI.convertPrecipitation_FromMetric(precipitationValue)) //$NON-NLS-1$
+            + String.format("%5s", roundDoubleToFloat(UI.convertPrecipitation_FromMetric(precipitationValue))) //$NON-NLS-1$
             + UI.UNIT_LABEL_DISTANCE_MM_OR_INCH;
 
       final String snowFall = Messages.Log_HistoricalWeatherRetriever_001_WeatherData_Snowfall
             + UI.SPACE
-            + String.format("%5s", UI.convertPrecipitation_FromMetric(snowFallValue)) //$NON-NLS-1$
+            + String.format("%5s", roundDoubleToFloat(UI.convertPrecipitation_FromMetric(snowFallValue))) //$NON-NLS-1$
             + UI.UNIT_LABEL_DISTANCE_MM_OR_INCH;
 
```

Comment:
Can you also display the atmospheric pressure in the hourly log ?

Reply:
Yes, fixed

![image](https://user-images.githubusercontent.com/25235665/158440124-cd390dd7-c8fd-4a26-8726-03730fb82054.png)